google_ngram_downloader/util.py

```diff
@@ -135,7 +135,14 @@ def iter_google_store(ngram_len, lang="eng", indices=None, verbose=False):
             sys.stderr.flush()
 
         request = session.get(url, stream=True)
-        assert request.status_code == 200
```

Comment:
This is a too general check. What if there is a network error and an existing file can't be downloaded.

I see two solutions here. Either to specify explicitly the language indices by copying them from the webpage, or by checking whether the HTTP status is 404 and then notifying a user explaining that not all character combinations are available in all languages. It would be nice to suggest the user to double check what have been downloaded with the list of available files.